src/Model/Review/Rating/SaveHandler.php

```diff
@@ -83,17 +90,20 @@ public function execute($entity, $arguments = [])
 
             if ($vote) {
                 $this->ratingFactory->create()
-                    ->setVoteId($vote->getId())
-                    ->setReviewId($reviewId)
-                    ->updateOptionVote($optionId);
+                                    ->setVoteId($vote->getId())
+                                    ->setReviewId($reviewId)
+                                    ->updateOptionVote($optionId);
             } else {
                 /** @var Rating $rating */
                 $this->ratingFactory->create()
-                    ->setRatingId($ratingId)
-                    ->setReviewId($entity->getId())
-                    ->addOptionVote($optionId, $entity->getEntityPkValue());
+                                    ->setRatingId($ratingId)
+                                    ->setReviewId($entity->getId())
+                                    ->addOptionVote($optionId, $entity->getEntityPkValue());
             }
         }
+        
+        $reviewObject = $this->reviewModel->load($reviewId);
+        $this->reviewResource->aggregate($reviewObject);
     }
```

Comment:
1. We can run aggregation in  class
`Divante\ReviewApi\Model\Review\Command\Save`
We already have review model there.
```
        $this->ratingSaveHandler->execute($dataModel);
        $this->reviewResource->aggregate($model);
```
2. To `Rating\SaveHandler` we can pass review model instead of review data object - we don't have to load review model again.